lib/config/config.go

```diff
@@ -56,6 +78,13 @@ func IsPerennialBranch(branchName string) bool {
 }
 
 
+func KnowsAllAncestorBranches(branchName string) bool {
+  return branchName == GetMainBranch() ||
```

Comment:
Thoughts about creating a helper method "IsMainBranch" to make this simpler and more expressive?